coordinator/coordinator_test.go

```diff
@@ -115,12 +115,12 @@ func newTestModules(t *testing.T) modules {
 	var bjj babyjub.PublicKeyComp
 	err = bjj.UnmarshalText([]byte("c433f7a696b7aa3a5224efb3993baf0ccd9e92eecee0c29a3f6c8208a9e81d9e"))
 	require.NoError(t, err)
-	coordAccount := &txselector.CoordAccount{ // TODO TMP
+	coordAccount := txselector.CoordAccount{ // TODO TMP
```

Comment:
I think, it will be better to give more descriptive todo message

Reply:
Agree, unfortunately this message wasn't added by me and I don't know what it's supposed to mean